main/src/ca/uwaterloo/flix/language/phase/Desugar.scala

```diff
@@ -18,7 +18,8 @@ package ca.uwaterloo.flix.language.phase
 
 import ca.uwaterloo.flix.api.Flix
 import ca.uwaterloo.flix.language.ast.DesugaredAst.Expr
-import ca.uwaterloo.flix.language.ast.{Ast, DesugaredAst, WeededAst}
+import ca.uwaterloo.flix.language.ast.WeededAst.ForFragment
```

Comment:
Drop import. We should always prefix with WeededAst.

Reply:
Thanks, I think IntelliJ did some “magic” for me.